packages/kit/src/nitro.ts

```diff
@@ -110,3 +110,21 @@ export function addServerImportsDir (dirs: string | string[], opts: { prepend?:
     config.imports.dirs[opts.prepend ? 'unshift' : 'push'](..._dirs)
   })
 }
+
+/**
+ * Add directories to be scanned by Nitro. It will check for:
+ *
+ * - `api/` folder (which will create API routes)
+ * - `middleware/` folder (which will register server middleware)
+ * - `utils/` folder (which will register auto-imports within your Nitro server)
```

Comment:
Same here for example. BTW do we plan to link JSDocs to official nuxt.com docs? Would be something to consider :D

Reply:
Yes, for sure 👌